src/devices/HardwareMonitor/OpenHardwareMonitor.cs

```diff
@@ -177,6 +177,11 @@ public IList<Sensor> GetSensorList()
                     string? identifier = Convert.ToString(sensor["Identifier"]);
                     string? parent = Convert.ToString(sensor["Parent"]);
                     string? type = Convert.ToString(sensor["SensorType"]);
+                    if (string.IsNullOrEmpty(name) || string.IsNullOrEmpty(identifier))
```

Comment:
this looks like unrelated change

Reply:
True, this was a fix for a bug I found from the sample application. Shall I put that into a separate PR?

Reply:
preferable separate PR (the missing copyright headers are ok though as they don't change the product)

Reply:
Was merged separtelly